cluster/images/etcd/Makefile

```diff
@@ -26,11 +26,11 @@
 # Except from etcd-$(tag) and etcdctl-$(tag) binaries, we also
 # need etcd and etcdctl binaries for backward compatibility reasons.
 # That binary will be set to the last tag from $(TAGS).
-TAGS?=2.2.1 2.3.7 3.0.17 3.1.11
-REGISTRY_TAG?=3.1.11
+TAGS?=2.2.1 2.3.7 3.0.17 3.1.11 3.2.14
+REGISTRY_TAG?=3.2.14
 ARCH?=amd64
 REGISTRY?=gcr.io/google_containers
-GOLANG_VERSION?=1.7.6
+GOLANG_VERSION?=1.9.2
```

Comment:
Can we add a comment/warning about go version with etcd version? Is there a site which indicates which is the officially supported version of goland for each version of etcd. (A table of version https://github.com/coreos/etcd/releases/ which shows etcd 3.3 comes from go 1.9.2, **etcd 3.2 comes from 1.8.5**, ...)

Reply:
There is no tabular version, but I can add a comment to this makefile referencing the release page you linked to.